src/controllers/actorController.js

```diff
@@ -36,45 +29,6 @@ const actorController = {
         });
       });
   },
-
-  getActorCharacters: (req, res, next) => {
```

Comment:
This was moved into scriptRepository since character information is most often used in the context of scripts.